src/components/user/auth/Login.jsx

```
@@ -38,10 +32,11 @@ export default function Login() {
             switch (data.user.roleId) {
               case 5:
                 console.log("redirect to admin");
-                navigate(ComponentPath.admin.dashboard);
+                var admin = ComponentPath.admin.dashboard;
+                navigate(admin);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)